src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -822,11 +829,13 @@ protected function isValidEntity($op, $entity_id) {
         // Just return FALSE, without an exception, for example when a list of
         // entities is requested, and we don't want to fail all the list because
         // of a single item without access.
-
         // Add the inaccessible item to the metadata to fix the record count in
         // the formatter.
         $inaccessible_records = $this->getMetadata()->get('inaccessible_records');
-        $inaccessible_records[] = $entity_id;
+        $inaccessible_records[] = array(
```

Comment:
I like the ideas, of making both configurable. Would you agree the default setting should be the strict ones (404 + no metadata)?